libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -139,7 +139,7 @@ export class TigerBackend implements IAnalyticalBackend {
     }
 
     public deauthenticate(): Promise<void> {
-        return Promise.resolve();
+        return this.authProvider.deauthenticate(this.getAuthenticationContext());
```

Comment:
In bear we check for the presence of the provider so that we can throw a better/more understandable error: https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-backend-bear/src/backend/index.ts#L268-L273 can you please do it here, too?